src/main/java/no/fdk/dataservicecatalog/dto/acat/TermsAndConditions.java

```diff
@@ -4,17 +4,19 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
+import java.util.Map;
+
 @Data
 @NoArgsConstructor
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class TermsAndConditions {
 
-    private String cost;
+    private String price;
 
-    private String usageLimitation;
+    private Map<String, String> usageLimitation;
 
-    private String performance;
+    private String capacityAndPerformance;
 
-    private String availability;
+    private String reliability;
```

Comment:
If I'm not mistaken, all fields in the class should be translatable strings, i.e. Map<String, String>